### PR TITLE
Upgrade version 3.0.0

### PR DESCRIPTION
**Pull request by dobladov**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v20.1.0
```
Comment (nitzanashi) [inline .nvmrc:1]: Why?

Comment (dobladov) [reply .nvmrc:1]: So you run it with `node20`, which is the version set for the runner https://github.com/zattoo/output/blob/main/action.yml#L18

Comment (nitzanashi) [reply .nvmrc:1]: I thought we update everything to 16 not 20, Web is still using 16

**Review by nitzanashi — APPROVED** [review]

**Comment by dobladov** [comment]
@z-merge it

**Pull request merged**